src/shared/components/lazyMobXTable/PaginatedDataStore.tsx

```diff
@@ -0,0 +1,140 @@
+import { action, computed, makeObservable, observable } from 'mobx';
+import { SortMetric } from 'shared/lib/ISortMetric';
+import {
+    DataFilterFunction,
+    getSortedData,
+    getSortedFilteredData,
+    getTableData,
+    getVisibileData,
+    ILazyMobXTableApplicationDataStore,
+} from 'shared/lib/ILazyMobXTableApplicationDataStore';
+import {
+    PageDirection,
+    PageStore,
+} from 'shared/components/lazyMobXTable/PageStore';
+
+export type PageParams = {
+    direction: PageDirection;
+    pageNumber: number;
+    pageSize: number;
+};
+
+export class PaginatedDataStore<T>
```

Comment:
make more clear that this this is server side pagination

Reply:
Added in [44ed633](https://github.com/cBioPortal/cbioportal-frontend/pull/4590/commits/44ed633252d40c7bb047f11fb4bc7838e07e7db7)